android-components/components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -120,7 +120,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:textSize="14sp"
-                    android:textColor="@color/photonRed70"
+                    android:textColor="?attr/textWarning"
```

Comment:
This should have been used instead so that we get a different color in dark mode.